package.json

```
@@ -151,8 +151,8 @@
     "typescript": "5.3.3",
     "url-loader": "^2.1.0",
     "vite": "5.3.2",
-    "vitest": "1.2.2",
-    "vitest-when": "0.3.1",
+    "vitest": "^2.1.8",
```

```suggestion
    "vitest": "2.1.8",
```
Maybe we should just lock the version just in case?